lessons/09-clientside-frameworks/README.md

```diff
@@ -1,51 +1,57 @@
-We are going to uncomment code to transform just one of the routes into an API endpoint and then use handlebars on the client side to render the content. 
+So you are all very familiar with server side templating using handlebars.
+In the Lesson 9 folder there is an in_class folder containing the cat app that you are all familiar with.
+We are going to uncomment code to transform just one of the routes into an API endpoint and then use handlebars on the client side to render the content.
 
-You can probably imagine that organizing a very complex application with dozens of pages and models can get really complicated and messy really fast. 
-Luckily, there are really fantastic publically available and free frameworks you can utilize to organize your application. 
-We will explore these in a bit. 
+You can probably imagine that organizing a very complex application with dozens of pages and models can get really complicated and messy really fast.
+Luckily, there are really fantastic publically available and free frameworks you can utilize to organize your application.
+We will explore these in a bit.
 
 ##MVC on the client
-We talked last class about MVC and how **awesome** it is. 
-Hopefully you all have an appreciation for the nice structure and separation of concerns it affords you. 
-When we restructure our application for the client side, we now move that organizational structure to the client side, as well. 
-The direct implementation will differ based on the specific framework, but in general you will organize your **views** as templates that you load in only once when the first page loads. 
-The **models** are organized javascript objects with some additional functionality, such as a link to an API endpoint that can be used to retrieve and update data. 
+We talked last class about MVC and how **awesome** it is.
+Hopefully you all have an appreciation for the nice structure and separation of concerns it affords you.
+When we restructure our application for the client side, we now move that organizational structure to the client side, as well.
+The direct implementation will differ based on the specific framework, but in general you will organize your **views** as templates that you load in only once when the first page loads.
+The **models** are organized javascript objects with some additional functionality, such as a link to an API endpoint that can be used to retrieve and update data.
 The **controller** is also on the client where it handles rendering the templates using the appropriate model data.
 
 ##Popular Frameworks
-It turns out that there are a TON of different javascript client-side frameworks to choose from for any project. 
+It turns out that there are a TON of different javascript client-side frameworks to choose from for any project.
 These frameworks have various strengths and weaknesses. In this class we are going to explore AngularJS and ReactJS, however, you are more than welcome to explore other templates on your own.
```

Comment:
Are we exploring both? I thought we decided on just doing React but I could be wrong